### ISSUE TITLE
update-dependencies script is failing at `npm audit fix`

### ISSUE THREAD

**Issue by tmatsuo** [googleapis/repo-automation-bots#4200]
Our `deps` workflow stopped creating PRs a while ago. I manually run the script and found out `npm audit fix` is failing. Let's try `--force` flag which allows major version update.

**Comment by SurferJeffAtGoogle** [comment]
Are there any negative consequences to allowing a major version update?

**Comment by tmatsuo** [comment]
It should be ok because as long as the test passes, we should be fine and the script won't create a PR if the test fails.

**Comment by SurferJeffAtGoogle** [comment]
Could you include the failure message in this bug report?

**Comment by tmatsuo** [comment]
This is an example workflow run:
https://github.com/googleapis/repo-automation-bots/actions/runs/2900715429

For example, auto-approve child job shows this:
```
================================================================
2022-08-22T01:39:54Z: Showing the failures 
================================================================
- [ ] auto-approve: 'npm audit fix' with ncu failed
- [ ] auto-approve: 'npm audit fix'  failed
```

When running `npm audit fix`, I get the following error:

```
added 75 packages, removed 10 packages, changed 26 packages, and audited 837 packages in 5s

81 packages are looking for funding
  run `npm fund` for details

# npm audit report

got  <11.8.5
Severity: moderate
Got allows a redirect to a UNIX socket - https://github.com/advisories/GHSA-pfrx-2q88-qq97
fix available via `npm audit fix --force`
Will install gts@4.0.0, which is a breaking change
node_modules/got
  package-json  <=6.5.0
  Depends on vulnerable versions of got
  node_modules/package-json
    latest-version  0.2.0 - 5.1.0
    Depends on vulnerable versions of package-json
    node_modules/latest-version
      update-notifier  0.2.0 - 5.1.0
      Depends on vulnerable versions of latest-version
      node_modules/update-notifier
        gts  <=3.1.0
        Depends on vulnerable versions of update-notifier
        node_modules/gts

5 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force
```

**Comment by tmatsuo** [comment]
`npm audit fix --force` will upgrade gts to 4.0.0 (as of today).

```
$ npm audit fix --force
npm WARN using --force Recommended protections disabled.
npm WARN audit Updating gts to 4.0.0,which is a SemVer major change.

added 10 packages, removed 75 packages, changed 17 packages, and audited 772 packages in 7s

76 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```
